src/Symfony/Component/Serializer/Tests/Normalizer/DateTimeNormalizerTest.php

```diff
@@ -280,7 +307,7 @@ public function testDenormalizeInvalidDataThrowsException()
 
     /**
      * @expectedException \Symfony\Component\Serializer\Exception\UnexpectedValueException
-     * @expectedExceptionMessage The data is either an empty string or null, you should pass a string that can be parsed with the passed format or a valid DateTime string.
+     * @expectedExceptionMessage The data is either an empty string or null, you should pass a string that can be parsed with the given format or a valid DateTime string.
```

Comment:
Again, shouldn't change since not the focus.

Reply:
restored.